wiki/Manual.md

```diff
@@ -897,52 +897,57 @@ Note: the `false` value for the `active` property doesn't always guarantee that
 
 Examples:
 
-1. `(market['limits']['amount']['min'] == 0.05) && (market['precision']['amount'] == 4)`
+1. `(market['limits']['amount']['min'] == 0.05) && (market['precision']['amount'] == 0.0001)`
 
   In this example the **amount** of any order placed on the market **must satisfy both conditions**:
 
-  - The *amount value* should be >= 0.05:
+  - The *amount value* must be >= 0.05:
     ```diff
     + good: 0.05, 0.051, 0.0501, 0.0502, ..., 0.0599, 0.06, 0.0601, ...
     - bad: 0.04, 0.049, 0.0499
     ```
-  - *Precision of the amount* should be up to 4 decimal digits:
+  - *Precision of the amount* can be up to 4 decimal digits (0.0001):
     ```diff
     + good: 0.05, 0.051, 0.052, ..., 0.0531, ..., 0.06, ... 0.0719, ...
     - bad: 0.05001, 0.05000, 0.06001
     ```
 
-2. `(market['limits']['price']['min'] == 0.019) && (market['precision']['price'] == 5)`
+2. `(market['limits']['price']['min'] == 0.019) && (market['precision']['price'] == 0.00001)`
 
   In this example the **price** of any order placed on the market **must satisfy both conditions**:
 
-  - The *price value* should be >= 0.019:
+  - The *price value* must be >= 0.019:
     ```diff
     + good: 0.019, ... 0.0191, ... 0.01911, 0.01912, ...
     - bad: 0.016, ..., 0.01699
     ```
-  - *Precision of price* should be 5 decimal digits or less:
+  - *Precision of price* can be up to 5 decimal digits (0.00001):
     ```diff
     + good: 0.02, 0.021, 0.0212, 0.02123, 0.02124, 0.02125, ...
     - bad: 0.017000, 0.017001, ...
     ```
 
-3. `(market['limits']['amount']['min'] == 50) && (market['precision']['amount'] == -1)`
+3. `(market['limits']['amount']['min'] == 50) && (market['precision']['amount'] == 10)`
 
   In this example **both conditions must be satisfied**:
 
-  - The *amount value* should be greater than or equal to 50:
+  - The *amount value* must be greater than or equal to 50:
     ```diff
     + good: 50, 60, 70, 80, 90, 100, ... 2000, ...
     - bad: 1, 2, 3, ..., 9
     ```
-  - A negative *amount precision* means that the amount should be an integer multiple of 10 (to the absolute power specified):
+  - A negative *amount precision* means that the amount should be an integer multiple of 10:
     ```diff
     + good: 50, ..., 110, ... 1230, ..., 1000000, ..., 1234560, ...
     - bad: 9.5, ... 10.1, ..., 11, ... 200.71, ...
     ```
 
-*The `precision` and `limits` params are currently under heavy development, some of these fields may be missing here and there until the unification process is complete. This does not influence most of the orders but can be significant in extreme cases of very large or very small orders.*
```

Comment:
everything is under heavy development and this comment is odd. but lmk if needed to place back